app/models/journeyDomain/transport/AuthorisationDomain.scala

```diff
@@ -40,12 +40,8 @@ case class AuthorisationDomain(authorisationType: AuthorisationType, referenceNu
   override def routeIfCompleted(userAnswers: UserAnswers, mode: Mode, stage: Stage): Option[Call] = Some {
     stage match {
       case AccessingJourney =>
-        if (userAnswers.get(AuthorisationTypePage(index)).isEmpty && index.isFirst) {
-          controllers.transport.authorisationsAndLimit.authorisations.index.routes.AuthorisationReferenceNumberController
-            .onPageLoad(userAnswers.lrn, mode, index)
-        } else {
-          controllers.transport.authorisationsAndLimit.authorisations.index.routes.AuthorisationTypeController.onPageLoad(userAnswers.lrn, mode, index)
-        }
+        controllers.transport.authorisationsAndLimit.authorisations.index.routes.AuthorisationReferenceNumberController
```

Comment:
So I believe doing this will make it impossible for the user to change the authorisation type. Again, I can see why it's been done, because sometimes we infer the authorisation type. We may need to have subtypes of the AuthorisationDomain where we can set this route accordingly. Happy to help out on that.

Reply:
This meets the requirements. User must delete an authorisation - they can't change the type. Marking as resolved.